src/client/utils/validations.js

```diff
@@ -180,14 +212,18 @@ function isValidVeteranAddress(data) {
       isValidField(isValidPhone, data.mobilePhone);
 }
 
+function isValidFinancialDisclosure(data) {
+  return isNotBlank(data.provideFinancialInfo.value);
+}
+
 function isValidSpouseInformation(data) {
```

Comment:
For the `isValidSpouseInformation` validation, do we want to make the spouse fields only validate `ifNotBlank` when the user selects `married`?